config.yml

```diff
@@ -24,6 +24,7 @@ sample_file: data/samples.tsv
 # which samples should we execute the pipeline on?
 # Comment out this line if you want to run all samples in the sample file
 #SAMP_NAMES: [region1, region2]
+SAMP_NAMES: [test2, 52217East]
```

Comment:
I usually try my best not to commit any of the changes I've made to the `config.yml` file so that it's still understandable to somebody who's just starting out with the pipeline.
For example, `region1` and `region2` are silly examples that I wrote about in [the documentation](https://github.com/beelabhmc/flower_map/wiki/Input-Data).

However, if you decide to create a new config parameter (like you did on [line 75](https://github.com/beelabhmc/flower_map/pull/15/files#diff-5a7db8dbadaef1b1b5a8738ba70b5ffac82b8e243732154165911284e08aad4bR75), for example), I would commit that.